github_pr.py

```diff
@@ -237,7 +237,7 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if (('numberonly' in args) and not args['numberonly']) or (('noratelimit' in args) and not args['noratelimit']):
+    if not args['numberonly'] and not args['noratelimit']:
```

Comment:
this was how it was working before, we need a quick fix. So we went back to what was working. I have another check that will address these issues that i am working on. This was breaking a build.